src/test/java/WorkoutAlgorithmTest.java

```diff
@@ -136,7 +136,7 @@ public void testCreateWorkoutWithAllEquipment()
         Assertions.assertFalse(workout.getExercises().isEmpty());
         workout.getExercises().forEach(exercise ->
             {
-            Assertions.assertEquals(Collections.singletonList(BodyPart.Chest), exercise.getExercise().bodyParts);
+            Assertions.assertTrue(exercise.getExercise().bodyParts.contains(BodyPart.Chest));
             Assertions.assertTrue(exercise.getExercise().equipment.contains(Equipment.Bodyweight) || exercise.getExercise().equipment.contains(Equipment.Barbell));
```

Comment:
Not your fault, but the test doesnt really reflect its name. The test method is called createWorkoutWithAllEquipment, but there is only two. Change this plz :)